src/test/java/com/rybka/service/exchange/ExchangeServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package com.rybka.service.exchange;
+
+import com.rybka.configuration.ExchangeProperty;
+import com.rybka.constant.ParameterValue;
+import com.rybka.model.ExchangeResponse;
+import com.rybka.model.TopCurrencyData;
+import com.rybka.service.connector.ExchangeRateConnector;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+class ExchangeServiceTest {
```

Comment:
The same comment as in the previous test. 

Formating, Constants.

Reply:
fixed